canonical-kubernetes/steps/05_logging/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+title: Choose Logging Application
+description: Choose a logging mechanism
+viewable: True
+additional-input:
+  - label: Logging
+    key: LOGGINGPLUGIN
+    type: choice
+    default: None
```

Comment:
@battlemidget i'm not sure if this is the best way to do this.  For both logging and monitoring, there is only 1 option, but I don't want that to be the default.  So i put this `None` thing that acts as a no-op and sets a generic result message in `after-deploy`.

Is there a better way to present a single option that I don't want checked by default?

Reply:
I think None is ok here, however, we should come up with an input option for checkboxes that would be better suited. http://urwid.org/reference/widget.html#checkbox @johnsca thoughts?

Reply:
Are you expecting other options to be added?  Does choosing this replace a built-in option, or is it strictly an additional component?

If the answers are "no" and "strictly additional," then there's a `boolean` [input type](https://github.com/conjure-up/conjure-up/blob/master/conjureup/ui/widgets/step.py#L22) that seems better suited.  Then, instead of labeling it as "Choose a logging mechanism," you'd label it as something like "Add logging components?"

Reply:
(Boolean type is also mentioned here: https://github.com/conjure-up/conjure-up/wiki/Spell-Step-Specification)